internal/extramanifest/extramanifest.go

```diff
@@ -130,6 +132,19 @@ func (h *EMHandler) ExportExtraManifestToDir(ctx context.Context, extraManifestC
 
 // ExtractAndExportManifestFromPoliciesToDir extracts CR specs from policies. It matches policies and/or CRs by labels.
 func (h *EMHandler) ExtractAndExportManifestFromPoliciesToDir(ctx context.Context, policyLabels, objectLabels map[string]string, toDir string) error {
+	// TODO fake the crd client instead of skipping it for UT
+	if h.CrdClient != nil {
+		_, err := h.CrdClient.ApiextensionsV1().CustomResourceDefinitions().Get(ctx, "policies.policy.open-cluster-management.io", metav1.GetOptions{})
```

Comment:
Quesiton, why cannot use the existing client to query CRD? The RBAC has the role

Reply:
Followed an old example ... Updated